src/snakefile

```diff
@@ -1,31 +1,37 @@
 
 configfile: "../data/config.yaml"
 
-options = [x[9:-3] for x in config["files"]] # extract out only the option path
-print(options)
+options = [x[9:-9] for x in config["files"]] # extract out only the option path
 
 rule all:
     input: 
-        expand("../steps/{option}.sol", option=options)
+        expand("../steps/{option}/model.sol", option=options)
     
 rule solve_lp:
     message:
         'Solving model via {config[solver]}...'
     input:
-        lp_file = "../steps/{option}.lp"
+        lp_file = "../steps/{option}/model.lp"
     output:
-        sol_file = "../steps/{option}.sol"
+        sol_file = "../steps/{option}/model.sol"
     params:
-        ilp = "../steps/{option}.ilp"
+        ilp = "../steps/{option}/model.ilp"
     log:
-        log = "../logs/solves/{option}.log"
+        log = "../logs/solves/{option}/model.log"
     shell: 
         """
         if [ {config[solver]} = gurobi ]
         then
           gurobi_cl Method=2 ResultFile={output.sol_file} ResultFile={params.ilp} {input.lp_file}
-        else
+        elif [ {config[solver]} = cbc ]
+        then
           cbc {input.lp_file} solve -sec 1500 -solu {output.sol_file}
```

Comment:
What do the 1500 sec do? Is it waiting time after the model is solved to ensure the writing of the sol file is complete? Or should the solution process take maximum 1500sec?

Reply:
Sets max solve time. I removed in revision for simplicity